### DISCUSSION TITLE
Mapping Arturia MiniLab 3 to MiniDexed

### DISCUSSION THREAD

**Discussion by smczech** [probonopd/MiniDexed#668] [Q&A]
Has anyone successfully bypassed using an encoder by using an Arturia MiniLab 3? I got it to Navigate around using the main encoder-click on the MiniLab and using the Shift button to select and Shift-Encoder Click to go home, but can't figure out how to get anything else to do anything (knobs, drum pads, etc). I can only see different voices - I can't select them. 

I would love an exported user file I could import using Arturia MIDI Control Center. 

Thanks in advance!
Scott

**Comment by diyelectromusic** [comment]
I don't have a file you can just import, but using the "MIDI buttons" options in minidexed.ini you should be able to get the Arturia to simulate the various up/down/prev/next/home/back/select actions using its pads.

I don't know if that would do what you want?

Kevin

**Reply by smczech** [reply]
Thanks, but I was trying to do that. I mapped all the pads to the numbers I set in the ini file, but they weren't working. The only success I had was when I started using the shift and encoder button controls.

**Comment by blixabloxa** [comment]
Hi Scott,

This what I have in my minidexed.ini file. MIDI 36 to 43  for the pads left to right (I don't use every pad). It seems to work when I use my Minilab 3. I got this from https://www.youtube.com/watch?v=uFpr1-5iYFI (he uses another Arturia device, but I guessed that the MIDI would be the same for the Minilab 3).

Enzo

```
# MIDI Button Navigation
#  Specify MIDI CC to act as a button (0 = ununsed, so don't use CC 0)
#  NB: Off < 64 < ON
# CC channel: 0=OFF; 1-16 MIDI Ch; >16 Omni
# If MIDIButtonNotes>0 then treat MIDIButton numbers as MIDI
# Note numbers, triggered with NoteOn/NoteOff, not CC numbers.
MIDIButtonCh=10
MIDIButtonNotes=1
MIDIButtonPrev=36
MIDIButtonNext=37
MIDIButtonBack=38
MIDIButtonSelect=39
MIDIButtonHome=43
MIDIButtonPgmUp=41
MIDIButtonPgmDown=40
MIDIButtonTGUp=0
MIDIButtonTGDown=0
```

**Reply by smczech** [reply]
Thanks so much for the info! I will try this today!

**Comment by Spacepadrille** [comment]
What worked for me (minilab 3) is to put the value 17 to MIDIButtonCh=17, because you use the minilab on midi 1 for notes and for 10 or other for buttons, so you need omni channel

**Comment by GertSchepens** [comment]
I was looking at this too, but this merge request and the version in there fixes everything and uses the display.

https://github.com/probonopd/MiniDexed/pull/743

Great new feature!
